README.md

```diff
@@ -1,3 +1,23 @@
 ## opendata-drupal ![GitHub workflow](https://github.com/vrk-kpa/opendata-drupal/actions/workflows/create-release.yml/badge.svg)
 
 Drupal docker image for open data portal (avoindata.fi). 
+
+### Local development
+
+Local development environment is built using docker-compose. The files are in the ´local´ folder and the compose file is setup so that only `drupal` image is built, other images are pulled with `latest` tags.
+
+#### Build & run
+
+```bash
+# build frontend (in modules/ytp-assets-common/)
+npm install && gulp

```

Comment:
Maybe we should replace this with `npm install && npm run gulp` as then it would use gulp version that is installed via npm, and developer wouldn't need to install gulp globally.

Reply:
I'll do the proposed changes + delete docker-compose.yml setup files, leaving only the files relevant to building Docker image behind.

This is because we discussed about this with @beazil and concluded that local env setup should reside in the main repository.